tempesta_fw/http.h

```diff
@@ -139,7 +139,7 @@ typedef struct {
 	int		state;
 	int		_i_st;
 	int		to_read;
-	union {
+	struct {
 		unsigned long acc;
 		unsigned long eol;
 	} _tmp;
```

Comment:
Why do we need `_tmp`, why not to embed `acc` and `eol` directly to `TfwHttpParser`?